_templates/how-to.md

```diff
@@ -1,26 +1,72 @@
 ---
 title: [Character Limit: 255, Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. The article title should be a combination of the main problem statement and if applicable, the key environment statement. Also determines the title in the table of contents.]
-description: [Brief description of the How-To article]
-type: [HowTo]
-page_title: [Text that will populate the HTML file’s head:title attribute]
+description: [Character Limit: 255, Recommendation is to have <= 155 characters for a description as this can impact indexing by search engines like Google, Bing, etc. The description should be brief overview of the how-to article. It should exclude unnecessary phrases like "This article contains ..."]
+type: [how-to]
+page_title: [Text that will populate the HTML file’s head:title attribute. Normally, this should be the same as the title field.]
```

Comment:
I think that we should choose a convention here. We ether add extra information about type (KB) and service\product or we leave it simple. I personally think that we will benefit from having a page_title: My Article | Knowledge Base | Telerik Platform. Other opinions?

About the Normally part, I was told different on the KB meeting :). However, please share a sample description about the page_title field and I will be happy to change it.